flowetl.Dockerfile

```diff
@@ -24,10 +24,10 @@ ENV SOURCE_VERSION=${SOURCE_VERSION}
 ENV SOURCE_TREE=FlowKit-${SOURCE_VERSION}
 WORKDIR /${SOURCE_TREE}/flowetl
 
-COPY --chown=airflow . /${SOURCE_TREE}/
-
+COPY --chown=airflow:1 . /${SOURCE_TREE}/
```

Comment:
Why's the gid 1?

Reply:
Airflow image issue - it dropped the group in the v2 image. Might've been fixed in the rc1 revisiion - I'll check